src/request.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable no-console */
+'use strict';
+
+const http = require('http');
+
+const BASE = 'http://localhost:8800';
+const pathname = '/file/index.html';
+const href = BASE + pathname;
```

Comment:
Concatenating URLs like this can lead to mistakes. It's better to use a dedicated function to join URL parts, such as the URL API in Node.js or a library like 'url-join' or use interpolation